Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+# Latest golang image on apline linux
+FROM golang:1.17-alpine
```

Comment:
Just a note for here, since you use `go 1.22.4` in your `go.mod` file, you should have a similar version in your docker file.

Reply:
Yes, you are right… my bad I missed that spot.